release-process-documentation/release-team-guides/ci-signal-playbook.md

```diff
@@ -1,41 +1,141 @@
+- Copy over any open test issues from previous release (ask previous CI Signal lead for the tracker sheet) and follow up on them with owners
+- Monitor [master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) and [master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) dashboards **twice a week** and ensure all failures and flakes are tracked via open issues
+  - Make sure all open issues have a kind/bug, priority/flake or priority/failing-test, priority/important-soon labels 
+  - Make sure the issue is assigned against the current milestone 1.x, using /milestone
+  - Assign the issue to appropriate SIG using /sig label
+  - CC the release manager and bug triage lead
+  - Add @kubernetes/sig-foo-test-failures to draw SIG-foo’s attention to the issue
+  - Post the test failure in SIG’s Slack channel to get help in routing the issue to the rightful owner(s)
+  - [Sample test failure issue](https://github.com/kubernetes/kubernetes/issues/63611)
+- Build and maintain a document of area experts / owners across SIGs for future needs e.g.: Scalability experts, upgrade test experts etc
+  
+#### **_Best Practice:_** 
+The SLA and involvement of signal lead at this stage might vary from release to release (and the CI Signal lead). However in an effort to establish an early baseline of the test health the signal lead can take an initial stab at the tests runs at the start of the release, open issues, gather and report on the current status. Post that, it might suffice to check on the tests **twice a week** due to high code churn and expected test instability.
 
-- quirk: if a job is listed as FAILING but doesn't have "Overall" as one of its ongoing failures, it's not actually failing
-- if a job is failing in one of the meta-stages (Up, Down, DumpClusterLogs, etc), I find the owning sig
-- if a job is failing because a specific test case is failing, and that test case has a [sig-foo] in its name, I go bug sig-foo
-- with unit test case failures, I try to infer the sig from the path, or OWNERS files in it, otherwise I find the owning sig to help
-- with verify failures, I try to infer the failure from the log, otherwise I find the owning sig to help
-- if a test case is failing in one job consistently, but not others, both the job owner and test case owner are responsible for identifying why this combination is different
+### Feature Freeze to Code Slush
+
+Day to day tasks remain pretty much the same as before, with the following slight changes
+- Monitor [master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) dashboard **daily**
+- Monitor [master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) **every other day**
+- Starting now, curate a **weekly CI Signal report** escalating current test failures and flakes and send to kubernetes-dev@googlegroups.com community. [See below](#reporting-status) for more details on the report format and contents
```

Comment:
Note here that increased attention on maintaining signal at this point in the cycle is rewarded by less stress during code slush and freeze?

Reply:
+1